<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Viewbooklist</class>
 <widget class="QMainWindow" name="Viewbooklist">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="verticalSpacing">
     <number>20</number>
    </property>
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <property name="spacing">
       <number>15</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="styleSheet">
           <string notr="true">color:rgb(111, 76, 19);
font: 12pt &quot;Segoe UI&quot;;</string>
          </property>
          <property name="text">
           <string>Search:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit">
          <property name="styleSheet">
           <string notr="true">color:rgb(111, 76, 19);
font: 12pt &quot;Segoe UI&quot;;</string>
          </property>
          <property name="placeholderText">
           <string>Enter book title</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label">
          <property name="styleSheet">
           <string notr="true">color:rgb(111, 76, 19);
font: 12pt &quot;Segoe UI&quot;;</string>
          </property>
          <property name="text">
           <string>Genre:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_gserach">
          <property name="minimumSize">
           <size>
            <width>160</width>
            <height>0</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">color:rgb(111, 76, 19);
font: 12pt &quot;Segoe UI&quot;;</string>
          </property>
          <item>
           <property name="text">
            <string>All</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>History</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Science</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Mystery</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Horror</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Business</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Cooking</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Biography</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Children</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Travel</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <property name="spacing">
         <number>2</number>
        </property>
        <item>
         <widget class="QPushButton" name="pushButton_3">
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton">
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="QTableView" name="tableView">
      <property name="styleSheet">
       <string notr="true">color:rgb(111, 76, 19);
font: 700 10pt &quot;Segoe UI&quot;;
</string>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>75</number>
      </attribute>
     </widget>
    </item>
    <item row="2" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>20</number>
      </property>
      <item>
       <widget class="QPushButton" name="Addbook">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">color:rgb(111, 76, 19);
font: 12pt &quot;Segoe UI&quot;;</string>
        </property>
        <property name="text">
         <string>Add new book</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="editbook">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">color:rgb(111, 76, 19);
font: 12pt &quot;Segoe UI&quot;;</string>
        </property>
        <property name="text">
         <string>Edit book details</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_2">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">color:rgb(111, 76, 19);
font: 12pt &quot;Segoe UI&quot;;</string>
        </property>
        <property name="text">
         <string>Insert book to member</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_deleteBook">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">color:rgb(111, 76, 19);
font: 12pt &quot;Segoe UI&quot;;</string>
        </property>
        <property name="text">
         <string>Delete book</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
